x/ibc-hooks/contracts/crosschain-swaps/README.md

```diff
@@ -74,152 +79,240 @@ To use this contract for crosschain swaps, the following are needed:
  * The chain needs a wasm execute middleware that executes a contract when
    receiving a wasm directive in the memo.
  * The swaprouter contract should be instantiated
- 
-Optional:
 
-This contract can be configured to track the acks or timeouts of the sent
-packets. For that we needed:
 
- * The chain to provide a way to call a contract when the ack or timeout for a
-   packet is received. 
- * Instantiate the contract with the `track_ibc_sends` set to true. Defaults
-   to false.
+## Testing
+  
+### Manual testing with localrelayer
 
+This guide will walk you through the process of manually testing the cross-chain swapping functionality in Osmosis.
 
-## Testing
+To test this contract, we will use the 
+[localrelayer tool](https://github.com/osmosis-labs/osmosis/tree/main/tests/localrelayer). This tool will setup two 
+osmosis testnets and a relayer between them.
 
-The following is the procedure to test the contracts on a testnet.
 
-### Requirements
+#### Prerequisites
+* osmosisd command line tool installed and configured.
+* [jenv](https://github.com/nicolaslara/jenv) command line tool installed, if you don't have it you can generate the json manually or modify the commands accordingly.
+* jq command line tool installed.
+* localrelayer tool installed, configured and running.
 
-You will need: 
+#### Setup
 
-* An osmosis testnet
-* Another testnet with an open channel to the osmosis testnet
-* Tokens on the non-osmosis testnet
-* A pool on the osmosis testnet between the IBC'd native asset of the
-  non-osmosis testnet and osmo
+Create an alias for chainA and chainB commands that will be used throughout the guide:
 
-### Localrelayer
+```bash
+alias chainA="osmosisd --node http://localhost:26657 --chain-id localosmosis-a"
+alias chainB="osmosisd --node http://localhost:36657 --chain-id localosmosis-b"
+```
```

Comment:
Wow good job on the docs, though def a bit painful we have to list this out all here.

Would be great to get into a standard localosmosis setup, or just directing people to a commented script that we actually use. (Rather than Readme + script)

Reply:
The script mostly works on its own, but there is some weirdness that make pool creation fail the first time and you have to kill it at that point and run it again. I didn't want to spend a lot of time debugging something that will run so seldom, so I just went with docs.

One downside of having a script is that people just trust it instead of going over the commands. If there's something wrong with the script it may not be caught

Reply:
oh, and... most people don't need to do this once the contracts are deployed (pools already exist)